doc/ref/api/index.md

```diff
@@ -17,6 +17,17 @@
    show
 ```
 
+## Sample Data
```

Comment:
Let's move this section to the bottom of the page. It's less important than plotting.